### PR TITLE
all: bump go to version 1.18

### PR DESCRIPTION
**Pull request by tobiaskohlbau**

### CODE REVIEW

**Review by BlackReloaded — CHANGES_REQUESTED** [review]
Typo in commit message "ping"

File: pkg/log/syslog/syslog.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//go:build !linux && !windows
```
[the remark this answers is not in the document]
Comment (tobiaskohlbau) [reply pkg/log/syslog/syslog.go:1]: This is not intended for linux but for every system not _linux_ and not _windows_ like for e.g. _darwin_. There exists an additional built constraint (done via filename) for linux: `syslog_linux.go`.

[the remark this answers is not in the document]
Comment (tobiaskohlbau) [reply pkg/log/syslog/syslog.go:1]: I just realised it's most likely a good idea to spin this into another commit.

[the remark this answers is not in the document]
Comment (tobiaskohlbau) [reply pkg/log/syslog/syslog.go:1]: Done

**Pull request merged**